web/text_layer_builder.css

```diff
@@ -18,12 +18,13 @@
   text-align: initial;
   inset: 0;
   overflow: hidden;
-  opacity: 0.25;
+  opacity: 1;
```

Comment:
Will this affect performance in any way, since looking at this patch I'm reminded of [bug 1054161](https://bugzilla.mozilla.org/show_bug.cgi?id=1054161)?

Reply:
In the mentioned bug, the patch relies on the text color being transparent, so it should fine.
I checked with the pdf in https://bugzilla.mozilla.org/show_bug.cgi?id=1813325 and the profiler and as far as I can tell I don't see any differences.